server/config/config.js

```diff
@@ -11,7 +11,8 @@ const config = {
     port: process.env.DB_PORT,
     PORT: 5000,
     dialect: 'postgres',
-    secret: process.env.JWT_SECRET
+    secret: process.env.JWT_SECRET,
+    logging: false
```

Comment:
why was this change required? Shouldn't we be logging in dev?

Reply:
I forgot to remove it, I wanted to use it to test something.